src/enemies_turn.s

```diff
@@ -0,0 +1,13 @@
+enemys_turn:
+	ld de,global_.enemies
+	ld c,n_enemies+1
+@loop:
+	ld h,d
+	ld l,e
+	call AI
```

Comment:
Ideally, this should be a macro because you only call it once instead of a function. Check #gb-dev for that. 
But it's fine for now I believe.

Reply:
well, I'm not sure about that. It's called for each enemy. And I don't think overusing the INCLUDE is a good thing (check #gb-dev too :wink:). This function is called in the first layer, and don't call any other function, so to me there is no problem using a function here.